protocol/messenger.go

```diff
@@ -915,9 +919,29 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 		}
 	}
 
+	if m.config.featureFlags.EnableNewsFeed {
+		// TODO get the last time we opened the app
```

Comment:
Relying on timestamps seems fragile. This approach breaks if the app launches without network connectivity. A more robust solution might be to persist the set of processed feed IDs instead (or if IDs aren't available, hashes of the content).

Reply:
Thanks, I'll keep that in mind when implementing it.
Feel free to comment any tips and tricks in the issue https://github.com/status-im/status-go/issues/6496